src/internet_identity/src/storage/tests.rs

```diff
@@ -21,38 +23,76 @@ fn should_match_actual_header_size() {
 fn should_report_max_number_of_entries_for_8gb() {
     let memory = VectorMemory::default();
     let storage = Storage::new((1, 2), memory);
-    assert_eq!(storage.max_entries(), 3774873);
+    assert_eq!(storage.max_entries(), 3_774_873);
 }
 
 #[test]
 fn should_serialize_header() {
     let memory = VectorMemory::default();
     let mut storage = Storage::new((1, 2), memory.clone());
-    storage.update_salt([5u8; 32]);
+    storage.update_salt([05u8; 32]);
     storage.flush();
 
     let mut buf = vec![0; HEADER_SIZE];
     memory.read(0, &mut buf);
-    assert_eq!(buf, hex::decode("49494301000000000100000000000000020000000000000000080505050505050505050505050505050505050505050505050505050505050505").unwrap());
+    assert_eq!(buf, hex::decode("494943030000000001000000000000000200000000000000001005050505050505050505050505050505050505050505050505050505050505050000020000000000000000000000000000000000000000000000").unwrap());
 }
 
 #[test]
-fn should_recover_header_from_memory() {
+fn should_recover_header_from_memory_v1() {
     let memory = VectorMemory::default();
     memory.grow(1);
-    memory.write(0, &hex::decode("494943010500000040e2010000000000f1fb09000000000000084343434343434343434343434343434343434343434343434343434343434343").unwrap());
+    memory.write(0, &hex::decode("494943010500000040e2010000000000f1fb090000000000000843434343434343434343434343434343434343434343434343434343434343430002000000000000000000000000000000000000000000000000").unwrap());
```

Comment:
The values did not actually change, it's just more data now. And in particular, we initialize the `entry_offset` to the value 512. Essentially this checks, that the `v1` header is still the same, additional values are `0` except for the one we expect to be initialized because it did not exist before.